<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectApp</class>
 <widget class="QMainWindow" name="SelectApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select App</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_paramGUI">
        <property name="text">
         <string>Set parameters</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_batchProcess">
        <property name="text">
         <string>Batch processing multiple files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_MWViewer">
        <property name="text">
         <string>Multi-worm tracker viewer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_SWViewer">
        <property name="text">
         <string>Single-worm tracker viewer</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>304</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
